samza-api/src/main/java/org/apache/samza/operators/data/MessageEnvelope.java

```diff
@@ -23,7 +23,7 @@
 
 
 /**
- * An entry in the input/output {@link org.apache.samza.operators.MessageStream}s.
+ * An entry in the input/output {@link org.apache.samza.operators.MessageStream}s
  */
 @InterfaceStability.Unstable
 public interface MessageEnvelope<K, M> {
```

Comment:
This is not yet removed completely. Still used in createInStream/createOutStream, etc. I plan to work on another patch trying to completely remove it from createInStream/createOutStream as well, by adding Function<M, K>, Function<M, V> to OutputStream constructor and BiFunction<K, V, M> to InputStream constructor.